js/main.js

```diff
@@ -3,19 +3,25 @@
 var CARDS_NUMBER = 25;
 var LIKES_MIN_NUMBER = 15;
 var LIKES_MAX_NUMBER = 200;
-var COMMENTS_MAX_NUMBER = 6;
+var COMMENTS_MAX_NUMBER = 30;
 var AVATARS_NUMBER = 6;
 
 var imageArray = [];
```

Comment:
неправильное имя, по критерию
```suggestion
var images = [];
```